csharp/LogicalRead.cs

```diff
@@ -10,6 +10,8 @@ namespace ParquetSharp
     public static class LogicalRead<TLogical, TPhysical>
         where TPhysical : unmanaged
     {
+        private const string UseDateTimeKindUnspecifiedSwitchName = "ParquetSharp.UseDateTimeKindUnspecified";
```

Comment:
You can make it `internal` instead of `private` so it can be used in tests as well.

Reply:
Personally not a fan of that because:
  1. You would have to access the variable with generics specified `LogicalRead<DateTime, long>.UseDateTimeKindUnspecifiedSwitchName`
  2. You will no longer be able to look at a unit test as a reference for how it can be implemented.

Reply:
There is static non-generic `LogicalRead` class that could be used for that, but I agree there are some befits of using raw strings in the test code. I don't have strong opinion here, I'll leave it to your judgment.